Godeps/_workspace/src/github.com/letsencrypt/go-jose/jose-util/main.go

```diff
@@ -22,7 +22,7 @@ import (
 	"os"
 
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/codegangsta/cli"
-	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/square/go-jose"
```

Comment:
Godep did this. Or to be more specific: When we ran Godep in the old location, it rewrote the original import to point at the vendorized copy. When I ran Godep right now it chose not to rewrite the import. My best guess: Godep is now smart enough to figure out that we don't actually use anything from main.go, and so chose not to do rewrites on it.